AutomatedTest/src/main/java/ca/bc/gov/open/staffnet/test/controllers/TestController.java

```diff
@@ -0,0 +1,75 @@
+package ca.bc.gov.open.staffnet.test.controllers;
+
+import ca.bc.gov.open.staffnet.test.services.TestService;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Map;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.core.ParameterizedTypeReference;
+import org.springframework.http.*;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.client.RestTemplate;
+import org.springframework.web.util.UriComponentsBuilder;
+
+@RestController
+@RequestMapping("/tests")
+public class TestController {
+    @Value("${test.host}")
+    private String host = "https://127.0.0.1/";
+
+    private final RestTemplate restTemplate = new RestTemplate();
+    private TestService testService;
+
+    @Autowired
+    public TestController(TestService testService) throws IOException {
+        this.testService = testService;
+        this.testService.setAuthentication("StaffnetIdentity-soapui-project-template.xml");
+        this.testService.setAuthentication("StaffnetBio-soapui-project-template.xml");
+    }
+
+    @GetMapping(value = "/all")
+    public ResponseEntity runAllTests() throws IOException {
+        UriComponentsBuilder builder = UriComponentsBuilder.fromHttpUrl(host + "tests/init");
+        try {
+            HttpEntity<Map<String, String>> resp =
+                    restTemplate.exchange(
+                            builder.build().encode().toUri(),
+                            HttpMethod.POST,
+                            new HttpEntity<>(new HttpHeaders()),
+                            new ParameterizedTypeReference<>() {});
+
+            if (resp.getBody().get("status").equals("fail")) {}
+
+        } catch (Exception ex) {
+
```

Comment:
also add a log and exit here pls

Reply:
I have added the catch statement. Can you please check now?